FrEee/Utility/JsonSerializer.cs

```diff
@@ -83,5 +84,37 @@ public string SerializeToString(object o)
 				dobj.InitializeData(ctx);
 			return JsonConvert.SerializeObject(kos);
 		}
+
+
+		static JsonSerializerSettings SimpleConverterSettings =  new JsonSerializerSettings
+			{
+				ConstructorHandling = ConstructorHandling.AllowNonPublicDefaultConstructor,
+				Formatting = Formatting.Indented,
+				MissingMemberHandling = MissingMemberHandling.Ignore,
+				ObjectCreationHandling = ObjectCreationHandling.Auto,
+				TypeNameHandling = TypeNameHandling.None,
+				TypeNameAssemblyFormat = System.Runtime.Serialization.Formatters.FormatterAssemblyStyle.Simple, 
+				ContractResolver = new JsonContractResolver()
+			};
+
+		/// <summary>
+		/// Simple Json serializer access. Use this for Script Json serialization. 
+		/// </summary>
+		/// <param name="obj"></param>
+		/// <returns></returns>
+		public static string SerializeObjectToJson(object obj)

```

Comment:
I think we can call this method `SerializeObject`; it's obvious from the name of the containing class that it serializes to JSON.

Reply:
Agreed.